Formula/allegro.rb

```diff
@@ -15,24 +15,18 @@ class Allegro < Formula
     sha256 "65a04aa3c0901264e54ca91f8982da085ea90bccabcb885fac054ba5219e19bd" => :mavericks
   end
 
-  devel do
-    url "http://download.gna.org/allegro/allegro-unstable/5.1.13.1/allegro-5.1.13.1.tar.gz"
-    sha256 "c6e6265c3d661d46270971b7fbf7db34c1873af62882a9ea4025ca1298edf14d"
-
-    depends_on "theora" => :recommended
-  end
-
   head do
-    url "https://github.com/liballeg/allegro5.git", :branch => "5.1"
-
-    depends_on "theora" => :recommended
+    url "https://github.com/liballeg/allegro5.git", :branch => "5.2"
```

Comment:
Now that there's only a `url` left in the `head do` block, please remove the `head do` block completely and instead put `head "https://github.com/liballeg/allegro5.git", :branch => "5.2"` below `url` and `sha256` of the stable version (leave an empty line between `sha256` and `head`).